.gitsecret/paths/mapping.cfg

```diff
@@ -1,2 +1,4 @@
 infra/vars/dev.tfvars:2ee02c007c23f230344e987fbc1f568eaac172877da7b6160bcd7e61e17d693d
 infra/vars/prod.tfvars:2ee02c007c23f230344e987fbc1f568eaac172877da7b6160bcd7e61e17d693d
+deploy/etc/opt/bfi/iiif-logging/secrets/mysql_password
```

Comment:
Do we need to create these and check them in ahead of time? We could just generate them on the deployment server and be done with it. By checking them in here I think we're also implying there's only set of MySQL credentials across any environment you want to provision (e.g. `DEV` / `PROD`).